spec/support/examples/imagebuilder.rb

```diff
@@ -208,6 +208,32 @@
     end
   end
 
+  context 'with a single volume specified' do
+    let(:args) do
+      {
+        from: from,
+        image_name: image_name,
+        volume: '/var/www',
+      }
+    end
+    it 'should expand the port to an array' do
+      expect(context).to include(expose: ['/var/www'])
```

Comment:
This should make an assertion against the `volume` attribute rather than `expose`